doc/nrf/known_issues.rst

```diff
@@ -785,3 +785,13 @@ NCSDK-6832: SMP Server sample fails upon initialization
 
 In addition to these known issues, check the current issues in the `official Zephyr repository`_, since these might apply to the |NCS| fork of the Zephyr repository as well.
 To get help and report issues that are not related to Zephyr but to the |NCS|, go to Nordic's `DevZone`_.
+
```

Comment:
Move this a few lines up, so that the comment in lines 784-787 is at the bottom.

Reply:
done